src/main/java/uk/gov/justice/digital/delius/controller/secure/DeliusDocumentsController.java

```diff
@@ -0,0 +1,45 @@
+package uk.gov.justice.digital.delius.controller.secure;
+
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestPart;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.multipart.MultipartFile;
+import uk.gov.justice.digital.delius.controller.advice.ErrorResponse;
+import uk.gov.justice.digital.delius.data.api.UploadedDocumentCreateResponse;
+import uk.gov.justice.digital.delius.service.DeliusDocumentsService;
+
+@RestController
+@RequestMapping(value = "secure", produces = MediaType.APPLICATION_JSON_VALUE)
+@AllArgsConstructor
+public class DeliusDocumentsController {
+
+    private DeliusDocumentsService deliusDocumentsService;
+
+    @PostMapping(path = "/offenders/{crn}/convictions/{convictionId}/document", consumes = MediaType.MULTIPART_FORM_DATA_VALUE)
```

Comment:
Other endpoints explicitly state the identifier type in a path part. This should probably be `/offenders/crn/{crn}/convictions/{convictionId}/document` for consistency